cmd/config.go

```diff
@@ -67,6 +67,9 @@ func newConfigCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVarP(
 		&stack, "stack", "s", "",
 		"The name of the stack to operate on. Defaults to the current stack")
+	cmd.PersistentFlags().StringVar(
```

Comment:
Nit, might be nice to have a central function that adds this flag. E.g. `addConfigFileFlag(&cmd)`.